blockchain/chainio_test.go

```diff
@@ -1000,12 +1002,12 @@ func TestUtxoSerialization(t *testing.T) {
 		// Ensure that the serialized bytes are decoded back to the expected utxo.
 		gotUtxo, err := deserializeUtxoEntry(test.serialized, test.txOutIndex)
 		if err != nil {
-			t.Errorf("serializeUtxoEntry #%d (%s): unexpected error: %v", i,
+			t.Errorf("deserializeUtxoEntry #%d (%s): unexpected error: %v", i,
```

Comment:
Nice catch.

Not something for this PR, but we probably should do a separate PR to update these to the newer style that just does `"...(%q: unexpected error: %v", test.name, err)`.  This style is a holdover from back before the test framework showed proper line numbers so the extra information was needed to pinpoint the failure, but that's no longer the case hence the style has changed.

Reply:
Yep, that makes sense.  I can update those separately.